### ISSUE TITLE
Ability to set a threshold of warnings to throw an error

### ISSUE THREAD

**Issue by Nitner0C** [realm/SwiftLint#696]
I've looked for this in the issues, but haven't found it...

It would be cool to set a warning threshold to raise an error

```yml
warning_threshold: 20
```

**Comment by jpsim** [comment]
This is fairly easy to build, so @Nitner0C I'd encourage you to try implementing this yourself if you're looking for ways to contribute 😄

**Comment by woodhamgh** [comment]
I would love this too!

@jpsim Could you point me in the right direction? Id love to look into contributing :)

**Comment by woodhamgh** [comment]
Ive prototyped something up, but it just counts the warnings thrown by swiftlint. 

Would you know if there is anyway we could pass the number of warnings currently in the project from xcode to swiftlint when we start linting. The reason is

Say we have 5 warnings in the project and then when we lint we get another 15 warnings I would like to throw an error as the total is 20 or above

@jpsim

**Comment by marcelofabri** [comment]
I think this can be closed as #757 was merged

**Comment by norio-nomura** [comment]
🙏
